app/controllers/games.js

```diff
@@ -98,3 +98,37 @@ export const getLeaderboard = (req, res) => {
     });
   });
 };
+
+/**
+ * Gets game log when game session ends
+ * @returns {object} description
+ * @export { function }
+ * @param {object} req
+ * @param {object} res
+ */
+export function getGameLog(req, res) {
+  // get game if user is authenticated
+  if (req.decoded) {
+    const userId = req.decoded._id;
+    Game.aggregate([
+      { $match: { userID: userId } },
+      {
+        $group: {
+          _id: '$gameID',
+          gameID: { $first: '$gameID' },
+          gameRound: { $first: '$gameRound' },
+          gameWinner: { $first: '$gameWinner' },
+          gamePlayers: { $first: '$gamePlayers' }
+        }
+      }
+    ])
+      .exec((err, gameLog) => {
+        if (err) {
+          return res.status(500).send({ err });
```

Comment:
should we just send the server error to the client? or do we want to make sure we send a user friendly error?

Reply:
Feedback will be implemented immediately